.gitignore

```diff
@@ -63,12 +63,34 @@ aequilibrae/paths/build/*
 docs/build/*
 docs/source/_generated/*
 docs/source/_auto_examples/*
+*.cpp 
```

Comment:
I am not sure about this.  @janzill ?

Reply:
These are cython generated, we switched to c++ when implementing path files and therefore there's a AoN.cpp that should be ignored